src/SFA.DAS.SharedOuterApi/Infrastructure/InternalApiClient.cs

```diff
@@ -21,19 +21,13 @@ public class InternalApiClient<T> : ApiClient<T>, IInternalApiClient<T> where T
             _azureClientCredentialHelper = azureClientCredentialHelper;
         }
 
-        protected override async Task AddAuthenticationHeader()
+        protected override async Task AddAuthenticationHeader(HttpRequestMessage request)
```

Comment:
Tedious I know, but the parameter name doesn't match the base class, making this a code smell according to static analysis tools....

Reply:
👍🏻 renamed and pushed!